ext/dapr-ext-workflow/dapr/ext/workflow/workflow_activity_context.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+from typing import Callable, TypeVar
+
+from durabletask import task
+
+T = TypeVar('T')
+TInput = TypeVar('TInput')
+TOutput = TypeVar('TOutput')
+
+
+class WorkflowActivityContext():
+
+    __ignore__ = "class mro new init setattr getattr getattribute"
+
+    def __init__(self, obj: task.ActivityContext):
+        self._obj = obj
+
+    def workflow_id(self) -> str:
+        return self._obj.orchestration_id
+
+    def task_id(self) -> int:
```

Comment:
Let's add the `@property` annotation to `task_id` and `workflow_id`.

Reply:
Done